_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -18066,13 +18063,6 @@
 /obj/effect/mapping_helpers/airlock/access/all/security/general,
 /turf/open/floor/plating,
 /area/station/maintenance/fore)
-"gKt" = (
```

Comment:
This spooked me but it looks like `eJX` had an identical definition to `cyW` and the same with `gKt` and `fbf`
so I guess the UpdatePaths script cleaned those up too even though they're not realted to fish at all